Dockerfile

```diff
@@ -20,18 +19,11 @@ RUN corepack enable
 RUN yarn set version 4.1.0
 
 # This will install dependencies for the sync-daemon, devDependencies included:
-RUN yarn workspaces focus sync-daemon
-
-RUN yarn workspace sync-daemon build
-
-# This will remove all dependencies and install production deps only:
-RUN yarn workspaces focus sync-daemon --production
+RUN yarn install
 
-FROM node:20-alpine
-
-WORKDIR /app
+RUN yarn workspace sync-daemon run build
 
-COPY --from=builder /app/packages/daemon/dist .
-COPY --from=builder /app/packages/daemon/node_modules ./node_modules
+# This will remove all dev dependencies and install production deps only
+RUN yarn workspaces focus -A --production
```

Comment:
I added the -A flag because otherwise we wouldn't be able to use wallet-lib as a peer dependency and would have issues with bitcore-lib

This is not installing dependencies from the wallet-service lambdas as we are ignoring its package in `.dockerignore`